libcomps/src/python/src/pycomps_dict.c

```diff
@@ -516,7 +516,7 @@ PyTypeObject PyCOMPS_DictIterType = {
     0,                          /* tp_clear */
     0,                          /* tp_richcompare */
     0,                          /* tp_weaklistoffset */
-    0,         /* tp_iter */
+    PyObject_SelfIter,          /* tp_iter */
```

Comment:
FWIW PyCOMPS_(M)DictIterType had PySeqIter_Type as tp_base, and the implicit 0 in tp_iter seems to inherit this anyway. But perhaps explicit is better than implicit.

Reply:
I've reverted this part to make the patch minimal.